gem/settings/base.py

```diff
@@ -316,8 +327,11 @@
 
 # Wagtail settings
 
+LOGIN_REDIRECT_URL = environ.get('LOGIN_REDIRECT_URL')
+LOGOUT_REDIRECT_URL = environ.get('LOGOUT_REDIRECT_URL')
+
+
 LOGIN_URL = 'molo.profiles:auth_login'
```

Comment:
Can you change this to `LOGIN_URL=reverse_lazy("oidc_authentication_init")`, please?